packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -494,12 +495,20 @@ async function getResourceManagementClientForArmDeployment(
 
 async function compileBicepToJson(
   bicepCommand: string,
-  bicepOrchestrationFilePath: string
+  bicepOrchestrationFilePath: string,
+  logger: LogProvider | undefined
 ): Promise<JSON> {
-  const command = `${bicepCommand} build "${bicepOrchestrationFilePath}" --stdout`;
   try {
-    const result = await Executor.execCommandAsync(command);
-    return JSON.parse(result.stdout as string);
+    const result = await executeCommand(
+      undefined,
+      logger,
+      { shell: false },
```

Comment:
Shell is used by default in exec(), if set shell false in spawn(), need to test it for some corner case. For example, bicep path contains special characters like this "/home/bicep/bin test1/bicep"

Reply:
`shell: false` is to deal with cases like these, because shell escape is not needed anymore.